t/data/openqa.ini

```diff
@@ -1 +0,0 @@
-../../etc/openqa/openqa.ini
```

Comment:
why?

Reply:
Because I had to add config values for testing instead of just testing with the default config. (We want to cover the part of the code that allows the users/admins to override values so we obviously need to supply this kind of configuration when running the test.)

Reply:
In other tests we just change the config on the fly to test specific features